src/input/readers/config/Config.cc

```diff
@@ -167,10 +166,12 @@ bool Config::DoUpdate() {
         return true;
     }
 
+    bool error = false;
```

Comment:
It does not seem like `error` is read? Can it be removed?

Reply:
Also, seeing the test change makes me think that for config, we should probably not suppress: There shouldn't be 1000s of lines in a config file and it's likely better for a user to see all the problematic ones.

Mind reverting? :-/

Reply:
Agreed :) and leaving `error` around was an oopsie.